app/controllers/issues_controller.rb

```diff
@@ -32,6 +39,12 @@ def create
     end
   end
 
+  def map_issue
+    @issue = Issue.find(params[:id])
+    @response = { issue: @issue, img: @issue.issue_attachments.first }
```

Comment:
response is bad name, it can be internal, and actually this variable is not needed, just pass hash to respond_with